specification/ml/put_job/MlPutJobRequest.ts

```diff
@@ -42,6 +42,10 @@ export interface Request extends RequestBase {
      */
     job_id: Id
   }
+  query_parameters: {
+    /** @deprecated 7.16.0 */
+    ignore_throttled?: boolean
```

Comment:
If it's true that this API supports ignore_throttled, it likely also supports the others mentioned here: https://www.elastic.co/guide/en/elasticsearch/reference/master/api-conventions.html#api-multi-index

Reply:
I thought `ignore_throttled` would only be supported as part of `indices_options` in the body. I am on mobile so can’t easily confirm now. Can someone from @elastic/ml-core check the code and confirm whether indices options are only supported in the body?

Reply:
Actually, this is on the job, not the datafeed so indices options don’t make sense at all. The indices options are in the body of the datafeed config.